scpiget.m

```diff
@@ -17,7 +17,8 @@
     type = 'char';
 end
 
-if ~contains(cmd, '?')
+%if ~contains(cmd, '?')
```

Comment:
No need to keep this line as a comment. You can delete it.

Reply:
@Palmitoxico, done!